about.html

```diff
@@ -0,0 +1,105 @@
+        <p class="event-email">Please contact us per the Email for any further Questions about Nairobi Innovation Week Conference 2022!</p>
+        <p class="email-line"><a href="#">uonbinnovation.ac.ke</a></p>
+      </div>
+    </section>
+    <section class="logo-make">
+        <h2>Nairobi Innovation Week 2022 Logo</h2>
+        <hr class="hr-logo">
+        <p class="para1">The logo for the 2022 Nairobi Innovation week was decided through the logo competition from 1 April to 10.April by the students of the University.After Online pulic voting ad Selection Committee process, the logo from Peter Nguka won the prize.After consultation with Mr Nguka we optimized the prize winning-logo for this year's conference.</p>
+
+        <p class="para2">The logo of the conference 2022 was decided through the logo competition.</p>
+
+        <div class="logo-img">
+          <img src="./imgs/niw2019.jfif" alt="Event Logo">
+        </div>
+    </section>
+
+    <section class="A-third">
+      <h3>See the past Nairobi Innovation Week</h3>
+      <hr>
+      <p>Take a look at the last two events which took place in Kisumu and Mombasa respectively.</p>
+      <div class="last-events">
+        <a href="https://www.youtube.com/watch?v=-M4SNHmqWG8"><img src="./imgs/photo18.jfif" alt=""></a>
+        <a href="https://www.youtube.com/watch?v=-yToP3YiR0I"><img src="./imgs/photo20.jfif" alt=""></a>
+      </div>
+    </section>
+
+    <section class="A-four">
+      <h2>Partner</h2>
+      <hr>
+      <div class="A-sponsors">
+        <img src="./imgs/kcb.png" alt="KCB bank">
+        <img src="./imgs/safaricom.png" alt="Safaricom">
+        <img src="./imgs/microsoft.png" alt="microsoft">
+        <img src="./imgs/udemy.png" alt="Udemy">
+      </div>
+    </section>
+  </main>
+
+
+  <footer class= "A-footer">
+    <div id="theme">
+      <!-- <p class="theme">NAIROBI INNOVATION WEEK</p> -->
+      <img src="./imgs/niw2019.jfif" alt="">
+    </div>
+    <div class="last_info">
+      <h4>2022 Nairobi Innovation Week, Some Rights Reserved</h4>
+      <p class="footer-para">The annual Nairobi Innovation Week (NIW) was launched in 2015 to promote Kenya’s and the world’s innovation and entrepreneurial ecosystems. Throughout the years, The University</p>
+    </div>
+
+  </footer>
```

Comment:
- [ ] Great job so far but why do you use different logos on the About and Home page? I think your footer should be the same and should have the same logo on those two pages since it's the same webpage, kindly change fix. :thanks :innocent::upside_down_face:

- Home Page:

![fothom](https://user-images.githubusercontent.com/79058364/170529671-27d18f89-ef17-420e-b54e-d27fa8142a67.png)


- About Page:

![fotab](https://user-images.githubusercontent.com/79058364/170529861-da512ebc-4709-4d30-97c0-d22204c415a9.png)